packages/hash/api/src/graphql/resolvers/index.ts

```diff
@@ -131,6 +132,7 @@ export const resolvers = {
     verifyEmail,
     sendLoginCode,
     loginWithLoginCode,
+    executeDemoTask,
```

Comment:
I realise this isn't important for the purposes of this demo task, but just want to make sure you're aware, in the "How to test this?" section of the PR description you indicate we have to login to the frontend to run the `executeDemoTask` mutation. That's not actually true, because you're not using any auth middleware here. If you wanted to make this mutation only available to authenticated users (who have signed up to HASH) this would become:
```suggestion
    executeDemoTask: loggedInAndSignedUp(executeDemoTask),
```

Reply:
That's a very helpful thing to highlight, thank you.
I got mislead by some early GraphQL authentication errors when I ran `yarn dev` (from the other endpoints I presume) and sort of assumed it was on all of the GraphQL endpoints.

Not something we need for the Demo task but I think all other ones should require authentication.